sdcm/tester.py

```diff
@@ -326,6 +326,19 @@ def run(self, result=None):
         result = super(ClusterTester, self).run(self.result)
         return result
 
+    @property
+    def reliable_replication_factor(self) -> int:
+        """
+        Calculates reliable replication factor for the test to run a load on
+        :return:
+        """
```

Comment:
This docstring doesn't clarify anything and it's considered useless.

Reply:
Added more details